sdcm/utils/url_validate.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+import requests
+
+SCYLLA_URL_REGEX = re.compile('.*(?P<url>http.*scylladb.com.*).*')
+URL_REGEX_AND_FORMAT_LIST = [
+    (re.compile(r'baseurl=(?P<url>http.*scylladb.com.*)'), '{url}repodata/repomd.xml'),
+    (re.compile(r'deb\s+\[arch=(?P<arch>.*?)\]\s(?P<url>http.*?)\s(?P<version_code_name>.*?)\s(?P<component>.*?)$'),
+     "{url}/dists/{version_code_name}/{component}/binary-{arch}/Packages"),
+    (re.compile(r'deb\s+(?P<url>http.*?)\s(?P<version_code_name>.*?)\s(?P<component>.*?)$'),
+     "{url}/dists/{version_code_name}/{component}/binary-amd64/Packages.gz"),
+]
+URL_ENV_VARIABLES = {
+    '$basearch': 'x86_64',
+    '$releasever': '7',
+}
+
+
+def verify_url_is_valid(url):
```

Comment:
name of the function and package is misleading
URL is a too broad term, this function doesn't check general URL but URLs in the repo files...

Reply:
Done, replace the method to `verify_scylla_file_version_is_valid`